### PR TITLE
add method getxpectedValues

### PR DESCRIPTION
**Pull request by MariaMoroz**
changed files:
 expected.json
 testData.json
 helper/methods.js

### CODE REVIEW

**Review by SergeiKuznetsov — APPROVED** [review]

File: data/expected.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
   "eventsChain1": {
-    "expectedValues": ["1", "2", "4", "8", "16", "32",
-      "64", "128", "256", "512"],
+    "expectedValues": ["4194303", "8388606", "16777212", "33554424",
+      "67108848", "134217696", "268435392", "536870784", "1073741568", "2147483136"],
     "header3And4": "Events Chain 1",
-    "f1Value": "1",
-    "f10Value": "512",
+    "f1Value": "4194303",
+    "f10Value": "2147483136",
```
Comment (Burdishka) [inline data/expected.json:13]: why not expectedValues[0] and expectedValues[9]? so after that, you don't need these two variables
"f1Value" and "f10Value"

File: data/expected.json [hunk 2]
```diff
@@ -6,11 +6,11 @@
     "oneRecord": 1
   },
   "eventsChain1": {
-    "expectedValues": ["1", "2", "4", "8", "16", "32",
-      "64", "128", "256", "512"],
+    "expectedValues": ["4194303", "8388606", "16777212", "33554424",
+      "67108848", "134217696", "268435392", "536870784", "1073741568", "2147483136"],
```
Comment (Burdishka) [inline data/expected.json:10]: so if you use the method getExpectedValues(value) that create expected result why do you need this expectedValues?

File: helper/methods.js [hunk 3]
```diff
@@ -5,6 +5,20 @@ export function userLogin(username, password) {
     $(selectors.login.passwordIF).setValue(password);
     $(selectors.login.singInBtn).click();
 }
+export function getExpectedValues(value){
+
+       const expectedValues = [];
+        let  valueInt = parseInt(value);
+           if(valueInt >= 0 && valueInt <= 4194303){
```
Comment (Burdishka) [inline helper/methods.js:12]: why 4194303?
what about if we gonna test with 4194304?

**Review by Burdishka — COMMENTED** [review]
Your code need Option+Command+L
I see in some lines two white spaces and etc

File: helper/methods.js (hunk 3 above)
Comment (MariaMoroz) [reply helper/methods.js:12]: fixed everything above except this. 
4194303 - is the maximum of integers which works in entity Chain1. Everything more won't work and we will get the message: "error saving entity".

Comment (Burdishka) [reply helper/methods.js:12]: Ok, cool. Thanks for explaining

**Review by Burdishka — APPROVED** [review]

**Review by SamJulia — APPROVED** [review]

**Pull request merged**